### ISSUE TITLE
Neural Network isn't how children learns

### ISSUE THREAD

**Issue by Chillance** [microsoft/AI-For-Beginners#185]
"We can construct a so-called artificial neural network inside a computer, and then try to teach it to solve problems by giving it examples. This process is similar to how a newborn child learns about his or her surroundings by making observations."

Uh, no. Neural networks are inspired by how neurons in our brain works internally, with A LOT of connections between different neurons, the same as layers in a neural network. But it's not how children learns. How this works with neural networks is that you give it A LOT OF DATA with specified connections and it builds an internal representation getting closer and closer to the result you expect. Also known as a model.

Children doesn't learn like that. A child learns more with smaller steps (it's called "baby steps" for a reason), by filtering out a lot of noise and focusing on single things and build up understanding of the world that way. Just like any other human learns things really. You don't learn a language by spamming data, you learn it step by step. Good luck feeding whole internet to a child and see how that goes regarding learning about things.

**Comment by Heritage200** [comment]
Okay

**Comment by nahaki** [comment]
thank you

**Comment by PRIYANKjakharia** [comment]
@Chillance can you specify which file you found the issue from?

**Comment by Chillance** [comment]
Oh, I don't recall. I searched for it and I couldn't find it anymore. So, seems it's removed.

I also used AI to give this little summation (it's an interesting read):

1. Learning Style:
**Neural Networks** (NNs) use **supervised learning** where they are trained on large amounts of labeled data. They adjust internal parameters based on errors using algorithms like backpropagation to minimize those errors. Learning is often narrow and task-specific.
**Children**, on the other hand, learn in a more **unsupervised and interactive manner**. They pick up information from diverse experiences and social interactions, often with minimal guidance. They explore, imitate, and learn from feedback in a more holistic, dynamic, and self-directed way, integrating multiple sources like sight, sound, touch, and social interaction.

2. Generalization and Transfer Learning:
**Neural Networks** are typically **narrow AI**, meaning they are designed for specific tasks and struggle to generalize across unrelated domains without retraining or reconfiguration.
**Children**, however, demonstrate strong **transfer learning**. They can apply knowledge from one context to another without explicitly being taught how to do so, due to their cognitive flexibility. For example, a child learning language can generalize grammatical rules to new sentences they have never heard before.

3. Memory and Forgetting:
**Neural Networks** retain everything they are trained on unless specific techniques like **weight pruning** or **memory replay** are employed. They do not naturally forget or reorganize knowledge in the way humans do.
**Children’s brains** reorganize information, emphasizing important experiences while forgetting irrelevant ones. Their learning is more about building understanding and models of the world, rather than just memorizing data.

4. Embodiment and Sensory Input:
**Neural Networks** process data (such as images, text, or sound) in a **disembodied** manner, without direct interaction with the physical world.
**Children’s** learning is deeply **embodied**, meaning they learn through physical interaction with the world. They touch, manipulate, and observe objects and actions. They also engage with the world socially, which plays a huge role in cognitive development
